tic_tac_toe/game.py

```diff
@@ -33,7 +48,14 @@ def _board_is_full(board):
 
     :param board: Game board.
     """
-    pass
+
+    # looks for "-" in every position in the board
+    # returns False if it finds one
+    for row in board:
+        if any(column for column in row if column == "-"):
```

Comment:
I understand what you're doing but found it a bit confusing. not sure whether there's a way to do this while still retaining its compact nature.